### ISSUE TITLE
Replace a Sweave Vignette With an R Markdown Vignette

### ISSUE THREAD

**Issue by villafup** [Bioconductor/sweave2rmd#63]
**Hello Outreachy applicants!**

_Note: This task can be completed by multiple Outreachy applicants_. Please choose this as your second (optional) contribution task to the Bioconductor Sweave2Rmd project as it builds upon the familiarity quiz in the Sweave2Rmd first contribution. 

**Prep Work**

* [Sweave2Rmd Knowledge Check](https://github.com/Bioconductor/sweave2rmd/issues/62)
* Please read our [contribution document](https://bioconductor.github.io/sweave2rmd/articles/contribute.html), which gives a good overview of the project plus the general workflow. In particular, you will be performing the tasks from [Create the R Markdown (.Rmd) vignette and Remove the Sweave vignette (.Rnw)](https://bioconductor.github.io/sweave2rmd/articles/contribute.html#create-the-r-markdown--rmd-vignette-and-remove-the-sweave-vignette--rnw).
* Install [pandoc](https://pandoc.org/)

**Contribution Task**

Make a pull request replacing a Sweave vignette with an R Markdown vignette.

1. Select a vignette from our [project board](https://github.com/orgs/Bioconductor/projects/2/views/6) that is labeled 'Easy' or 'Medium', set to 'Todo' and are unassigned. (As a FYI, although there is no maximum # of vignettes you're allowed to convert, project leads must review everyone's first PR before we are able to assess any additional vignettes from anyone contributing to the project). 
2. Write a comment with the path of the vignette you'd like to do--e.g., BiocParallel/vignettes/Introduction_To_BiocParallel.Rnw. We will write your GitHub username as 'Assigned' and set 'Last Modified' to today.
3. Perform the steps in [Create the R Markdown (.Rmd) vignette and Remove the Sweave vignette (.Rnw)](https://bioconductor.github.io/sweave2rmd/articles/contribute.html#create-the-r-markdown--rmd-vignette-and-remove-the-sweave-vignette--rnw). Note: You do not need to contact the maintainers to work on this set of vignettes as they have either been previously contacted OR they are maintained by the Bioconductor Core Team.
4. Make sure that you request a review by writing @Bioconductor/sweave2rmd in a comment so that we can review your pull request. For example, @Bioconductor/sweave2rmd please review my contribution for Outreachy.
5. Make a comment in this issue with a link to your PR.
6. If you receive any feedback requesting changes, please address them.

Tip: You can find the PDF you'll be replacing by visiting the landing page for the package under Documents. You can look up the package at [Bioconductor Software Packages](https://bioconductor.org/packages/devel/BiocViews.html#___Software).
Tip: You can read how we [review PRs](https://bioconductor.github.io/sweave2rmd/articles/contribute.html#review-a-pull-request) in the contribution document. Use the checklist to assess if your PR will pass review.

If your pull request is good, it will be merged into Bioconductor!

Remember to record your contributions in Outreachy at https://www.outreachy.org/outreachy-may-2023-internship-cohort/communities/bioconductor/convert-sweave-vignettes-to-r-markdown/contributions/.

**Comment by Khadeeejah** [comment]
I would be working on this vignette:`  genefilter/vignettes/howtogenefilter.Rnw` 

Thanks

**Comment by devbird007** [comment]
Hello, I would like to work on this vignette: genefilter/vignettes/howtogenefinder.Rnw

**Comment by sirjoka** [comment]
I would like to work on the following vignette - genefilter/vignettes/howtogenefinder.Rnw

**Comment by sirjoka** [comment]
I would like to work on the following vignette - graph/vignettes/graph.Rnw

**Comment by sonali8434** [comment]
Hello , I want to work in Rsamtools/vignettes/Rsamtools-Overview.Rnw please assign me this

**Comment by sonali8434** [comment]
Hello, I would like to work on this vignette:  Rsamtools/vignettes/Rsamtools-Overview.Rnw please assign me this.

**Comment by Reece11** [comment]
Hi @villafup 
I am working on this vignette:  **`GeneMeta/vignettes/GeneMeta.Rnw`** please assign me 
thank you

**Comment by mcarlsn** [comment]
hi hi all - quick plug that for those of you who are interested in getting started on Sweave2Rmd contribution tasks, please start with the first contribution task ([instructions here](https://github.com/Bioconductor/sweave2rmd/issues/62)). Once completed, you have the option of then starting the [second (optional) contribution task](https://github.com/Bioconductor/sweave2rmd/issues/63) where you claim a vignette from the project board and convert it. The first contribution task is designed as a ramp to the second task, which is why we ask that you work on them sequentially. Please let us know if you have any questions!

**Comment by jwokaty** [comment]
@mcarlsn Thanks for the suggestion. 

The first task will help set you up for success. Please consider doing it if you haven't already as we tried to highlight important parts of our work that are easily missed.

I've assigned everyone their requested vignettes. We'll update the board with more vignettes later in the week.

**Comment by kemicky** [comment]
Hi, please I am working on : 
GenomicAlignments/vignettes/summarizeOverlaps.Rnw
Kindly please assign it to me, thank you

**Comment by devbird007** [comment]
> Hello, I would like to work on this vignette: genefilter/vignettes/howtogenefinder.Rnw

I have taken the familiarity quiz and would now commit to the task assigned to me.

**Comment by MonalikaPatnaik** [comment]
Heyy.. I would like to work on the vignette: GenomicAlignments/vignettes/summarizeOverlaps.Rnw
Kindly assign this to me!

**Comment by kemicky** [comment]
@MonalikaPatnaik : That is the one I am working on just waiting for it to be assigned to me.

**Comment by sirjoka** [comment]
> I would like to work on the following vignette - graph/vignettes/graph.Rnw

I have submitted the Quiz for Task 1 and will be proceeding to complete Task 2

**Comment by MonalikaPatnaik** [comment]
> @MonalikaPatnaik : That is the one I am working on just waiting for it to be assigned to me.

Oh okayy..Are there any open issues to work on in the project board under outreachy label, I found all the issues have already been assigned. I have completed my first contribution and looking for the issue to contribute on!

**Comment by Reece11** [comment]
> I've assigned everyone their requested vignettes. We'll update the board with more vignettes later in the week.

i think this answers your querry [here](https://github.com/Bioconductor/sweave2rmd/issues/63#:~:text=I%27ve%20assigned%20everyone%20their%20requested%20vignettes.%20We%27ll%20update%20the%20board%20with%20more%20vignettes%20later%20in%20the%20week.)

**Comment by MonalikaPatnaik** [comment]
> 

Thanks @Reece11 for clarification!

**Comment by kemicky** [comment]
> > @MonalikaPatnaik : That is the one I am working on just waiting for it to be assigned to me.
> 
> Oh okayy..Are there any open issues to work on in the project board under outreachy label, I found all the issues have already been assigned. I have completed my first contribution and looking for the issue to contribute on!

Hi,I understand we can work on the same project , I am a novice here to am just getting around with the task, learning and understanding the task. so I guess it ok ..

**Comment by kemicky** [comment]
@MonalikaPatnaik : I will explain to you how we are to do the steps for how we are to complete task 1 when I am done. cheers.

**Comment by devbird007** [comment]
> > Hello, I would like to work on this vignette: genefilter/vignettes/howtogenefinder.Rnw
> 
> I have taken the familiarity quiz and would now commit to the task assigned to me.

Hi @jwokaty I have completed the task assigned to me and submitted a pull request for it. Next I would like to be assigned this task: `Category/vignettes/ChromBand.Rnw`

**Comment by jwokaty** [comment]
> > > Hello, I would like to work on this vignette: genefilter/vignettes/howtogenefinder.Rnw
> > 
> > 
> > I have taken the familiarity quiz and would now commit to the task assigned to me.
> 
> Hi @jwokaty I have completed the task assigned to me and submitted a pull request for it. Next I would like to be assigned this task: `Category/vignettes/ChromBand.Rnw`

Rather than start on a new task, I am going to ask you to review your work with https://bioconductor.github.io/sweave2rmd/articles/contribute.html#review-a-pull-request. I'll flag your PR for review, which may take a few days. You should address any changes made by the reviewer. This is especially important for the first PR so you don't make the same mistakes in subsequent PRs. Once it is merged, you can start a new task.

**Comment by sonali8434** [comment]
I would like to work on the following vignette - graph/vignettes/graph.Rnw please assign me this .

**Comment by jwokaty** [comment]
> I would like to work on the following vignette - graph/vignettes/graph.Rnw please assign me this .

Since graph is already assigned, I'll assign you to GenomicFiles/vignettes/GenomicFiles.Rnw as you discussed and remove you from Rsamtools/vignettes/Rsamtools-Overview.Rnw.

**Comment by iamatifmoin** [comment]
I will be working on: 
> ACME/vignettes/ACME.Rnw

Thanks.

**Comment by sonali8434** [comment]
Hi @jwokaty I have completed the  2nd task assigned to me GenomicFiles/vignettes/GenomicFiles.Rnw and submitted a pull request for it please review my PR link- https://github.com/Bioconductor/GenomicFiles/pull/7  and let me Know what to do next. 
Thanks ..

**Comment by sonali8434** [comment]
> > I would like to work on the following vignette - graph/vignettes/graph.Rnw please assign me this .
> 
> @sonali8434 Since graph is already assigned, I'll assign you to GenomicFiles/vignettes/GenomicFiles.Rnw as you discussed and remove you from Rsamtools/vignettes/Rsamtools-Overview.Rnw.

**Comment by sonali8434** [comment]
> > > I would like to work on the following vignette - graph/vignettes/graph.Rnw please assign me this .
> > 
> > 
> > @sonali8434 Since graph is already assigned, I'll assign you to GenomicFiles/vignettes/GenomicFiles.Rnw as you discussed and remove you from Rsamtools/vignettes/Rsamtools-Overview.Rnw.

Hi @jwokaty I have completed the 2nd task assigned to me GenomicFiles/vignettes/GenomicFiles.Rnw and submitted a pull request for it please review my PR link- https://github.com/Bioconductor/GenomicFiles/pull/7 and let me Know what to do next.
Thanks ..

**Comment by MonalikaPatnaik** [comment]
> 

Sure..Thankyou:)

**Comment by jwokaty** [comment]
I wanted to mention a few important things.

1. Please do the knowledge quiz at https://github.com/Bioconductor/sweave2rmd/issues/62. It will prepare you for this task and we will review the responses. You'll get feedback at the email associated with your response. This email should ideally match the one you used for Outreachy so we can verify you did it if you include it in your contributions.
2. Please only select vignettes from the project board at https://github.com/orgs/Bioconductor/projects/2/views/6. We selected these vignettes specifically because they're maintained by the Bioconductor Core Team; these packages and vignettes are not deprecated; etc., so that we can control merging the PR. If you pick a vignette that is not maintained by the core team, we can review the PR but we will ultimately have no control over merging it because it is up to the maintainer since the contribution is to their Bioconductor package. We will add more vignettes to the board soon.
3. If you finish this task, your first PR needs to go through review. This will take time, so be patient and proactive by using the checklist to perform your own review first: https://bioconductor.github.io/sweave2rmd/articles/contribute.html#review-a-pull-request. This way you won't make the same mistake on your second PR. You could also look at some vignettes in other packages, so that you start to see the general format of R Markdown, such as https://github.com/Bioconductor/BiocHubsShiny or https://github.com/Bioconductor/graph.

**Comment by jwokaty** [comment]
> I will be working on:
> 
> > ACME/vignettes/ACME.Rnw
> 
> I have completed the first contribution task. Thanks.

@iamatifmoin Please choose a vignette from the board filtered for Outreachy: https://github.com/orgs/Bioconductor/projects/2/views/6. We'll be adding more vignettes soon.

**Comment by jwokaty** [comment]
> > > > I would like to work on the following vignette - graph/vignettes/graph.Rnw please assign me this .
> > > 
> > > 
> > > @sonali8434 Since graph is already assigned, I'll assign you to GenomicFiles/vignettes/GenomicFiles.Rnw as you discussed and remove you from Rsamtools/vignettes/Rsamtools-Overview.Rnw.
> 
> Hi @jwokaty I have completed the 2nd task assigned to me GenomicFiles/vignettes/GenomicFiles.Rnw and submitted a pull request for it please review my PR link- [Bioconductor/GenomicFiles#7](https://github.com/Bioconductor/GenomicFiles/pull/7) and let me Know what to do next.
> 
> I can work on Category/vignettes/ChromBand.Rnw in the remaining time if you assign me this.
> 
> Thanks ..

@sonali8434 Please see suggestion 3 in https://github.com/Bioconductor/sweave2rmd/issues/63#issuecomment-1460834595.

**Comment by kemicky** [comment]
@jwokaty : Thank you everyone, Thank you for your clarification.
 If I may just clarify the project I initially choose to work for my first task : 
GenomicAlignments/vignettes/summarizeOverlaps.Rnw
The maintainer is not Bioconductor
 After your instructions update now, Do I need to change it to one that has Bioconductor has maintainer as per your instructions ?.
If Yes, Kindly advice and I am very happy to quickly change it.
Thank you all for your patience.

**Comment by kemicky** [comment]
@jwokaty: 
Please may you kindly update my assignment, I have chosen a new project maintained by Bioconductor Core Team as instructed and please remove me from the one I chose before .
Please may you kindly assign to me: **Category/vignettes/ChromBand.Rnw**
Thank you all

**Comment by iamatifmoin** [comment]
> > I will be working on:
> > > ACME/vignettes/ACME.Rnw
> > 
> > 
> > I have completed the first contribution task. Thanks.
> 
> @iamatifmoin Please choose a vignette from the board filtered for Outreachy: https://github.com/orgs/Bioconductor/projects/2/views/6. We'll be adding more vignettes soon.

@jwokaty Okay!

I'll be doing this one:

> GOstats/vignettes/GOstatsForUnsupportedOrganisms.Rnw

**Comment by sirjoka** [comment]
@Bioconductor/sweave2rmd please review my work. I created the RMarkdown file for the graph vignette and created a pull request for the changes made to my branch.

**Comment by jwokaty** [comment]
> Hi @villafup I am working on this vignette **geneplotter/vignettes/byChroms.Rnw** and **geneplotter/vignettes/visualize.Rnw**. Please assign me
> 
> Thank you

@Rohit-Satyam Can I ask you to select 1 to work on? We'll review the PR completely before you get another vignette.

**Comment by jwokaty** [comment]
If you finish your PR, please make a comment with the link to your PR. Thank you :)

**Comment by sirjoka** [comment]
> If you finish your PR, please make a comment with the link to your PR. Thank you :)

Hi, please here is the link to my Pull Request https://github.com/Bioconductor/graph/pull/15

**Comment by devbird007** [comment]
> If you finish your PR, please make a comment with the link to your PR. Thank you :)

This is my pull request here: https://github.com/Bioconductor/genefilter/pull/10

**Comment by sonali8434** [comment]
Hello,
 @jwokaty  my PR link is incorrect in  https://github.com/orgs/Bioconductor/projects/2/views/6  my correct PR link is https://github.com/Bioconductor/GenomicFiles/pull/8 please update the link in the project board. 
Thank you .

**Comment by sirjoka** [comment]
> If you finish your PR, please make a comment with the link to your PR. Thank you :)

Hi, please here is the link to my Pull Request https://github.com/Bioconductor/graph/pull/16

Please take note this is a new pull request. Thank you

**Comment by Daviskojob** [comment]
I would be working on this vignette
bioDist/vignettes/bioDist.Rnw

**Comment by Dbn-tech** [comment]
hello @jwokaty , i would like to work on this vignette geneplotter/vignettes/byChroms.Rnw

**Comment by SunSummoner** [comment]
Hii @jwokaty 
I would be working on geneplotter/vignettes/byChroms.Rnw
Thank you

**Comment by jwokaty** [comment]
@Rohit-Satyam Which vignette are you working on?

**Comment by jwokaty** [comment]
@Dbn-tech @SunSummoner I am trying to determine if someone is already working on this vignette, so you may have to choose other. Or maybe you want to go ahead and choose another so that you don't have to wait.

**Comment by kemicky** [comment]
Hello everyone:
Good day
**Please I am having file does not exist error  while using : pandoc -f latex -t markdown file.Rnw -o file.Rmd  in RStudio.
I have installed rmarkdown and latex (tinytex)**
Any suggestion please, thank you

**Comment by Dbn-tech** [comment]
hello again @jwokaty ...i would like to work on this GOstats/vignettes/GOstatsHyperG.Rnw as you solve the issue on who is working on the other vignette

**Comment by sirjoka** [comment]
> > If you finish your PR, please make a comment with the link to your PR. Thank you :)
> 
> Hi, please here is the link to my Pull Request [Bioconductor/graph#16](https://github.com/Bioconductor/graph/pull/16)
> 
> Please take note this is a new pull request. Thank you

Hi, my new PR is ready. https://github.com/Bioconductor/graph/pull/18
Thank you so much.

**Comment by jwokaty** [comment]
@kemicky It might be better to copy paste the command and the error or show a screenshot in slack. Can you put it there and "@" mention me so that I get notified when you post it?

**Comment by Khadeeejah** [comment]
Hi this is the link to my pr

https://github.com/Bioconductor/genefilter/pull/11

**Comment by kemicky** [comment]
> @kemicky It might be better to copy paste the command and the error or show a screenshot in slack. Can you put it there and "@" mention me so that I get notified when you post it?

@jwokaty :  ok

**Comment by Khadeeejah** [comment]
> Hi this is the link to my pr @jwokaty [Bioconductor/genefilter#11](https://github.com/Bioconductor/genefilter/pull/11)

@jwokaty

**Comment by jwokaty** [comment]
@SunSummoner I am sorry for the delayed response. I am trying to clarify with @Rohit-Satyam what vignette they are working on. If they are not working on the vignette you requested, I will give it to you; however, it may be better to select another vignette so that you can continue.

**Comment by SunSummoner** [comment]
Hi @jwokaty 
As you said about the uncertainty regarding whose working with the geneplotter vignette. I'd like to change my vignette and work with ShortRead/vignettes/Overview.Rnw.
Thank you

**Comment by bellatrick** [comment]
Hi, @jwokaty. I'd like to work on this vignette Rsamtools/vignettes/Rsamtools-Overview.Rnw

**Comment by jwokaty** [comment]
@bellatrick I'll assign you this vignette. It has an issue that I've discussed at https://community-bioc.slack.com/archives/C037LN1HQDB/p1678601431723939?thread_ts=1678564533.972929&cid=C037LN1HQDB.

**Comment by bellatrick** [comment]
> @bellatrick I'll assign you this vignette. It has an issue that I've discussed at https://community-bioc.slack.com/archives/C037LN1HQDB/p1678601431723939?thread_ts=1678564533.972929&cid=C037LN1HQDB.

Thank you @jwokaty ,I will go through the discussion

**Comment by sirjoka** [comment]
@jwokaty Good morning. Please do you mind reviewing my PR when you have the time. 
https://github.com/Bioconductor/graph/pull/18

Thank you

**Comment by sirjoka** [comment]
> @jwokaty Good morning. Please do review my PR when free. [Bioconductor/graph#18](https://github.com/Bioconductor/graph/pull/18)
> 
> Thank you

@jwokaty New PR https://github.com/sirjoka/graph/pull/1

Please review

**Comment by sirjoka** [comment]
> @jwokaty Good morning. Please do review my PR when free. [Bioconductor/graph#18](https://github.com/Bioconductor/graph/pull/18)
> 
> Thank you

I couldn't update the file using the old branch something about not able to locate it. I think because I used Git Bash and closed the session.  So I created a new PR and updated https://github.com/sirjoka/graph/pull/1
Please review. 
Thank you

**Comment by Rohit-Satyam** [comment]
> @SunSummoner I am sorry for the delayed response. I am trying to clarify with @Rohit-Satyam what vignette they are working on. If they are not working on the vignette you requested, I will give it to you; however, it may be better to select another vignette so that you can continue.

Sorry for late reply, I just pushed the changes for **geneplotter/vignettes/byChroms.Rnw** only as you asked to do only single vignette.

**Comment by jwokaty** [comment]
@BerylKanali Could I ask you to review https://github.com/Bioconductor/geneplotter/pull/2 for @Rohit-Satyam?

@Dbn-tech I have asked @villafup to review https://github.com/Bioconductor/GOstats/pull/2

**Comment by bellatrick** [comment]
@jwokaty , hi,  I opened a PR on [Rsamtools](https://github.com/Bioconductor/Rsamtools/pull/53)
Can I get a review whenever you're available?

**Comment by BerylKanali** [comment]
> @BerylKanali Could I ask you to review [Bioconductor/geneplotter#2](https://github.com/Bioconductor/geneplotter/pull/2) for @Rohit-Satyam?
> 
> @Dbn-tech I have asked @villafup to review [Bioconductor/GOstats#2](https://github.com/Bioconductor/GOstats/pull/2)

@jwokaty Sure.

**Comment by sonali8434** [comment]
@jwokaty  Please assign me GOstats/vignettes/GOvis.Rnw from project board. 
Thank you.

**Comment by sopeadeniji** [comment]
@jwokaty  Please assign me geneplotter/vignettes/visualize.Rnw on project board. Thanks.

**Comment by Dbn-tech** [comment]
> Hello @jwokaty, i have completed the second contribution and I would like you to review it and advise where necessary. [Bioconductor/GOstats#2](https://github.com/Bioconductor/GOstats/pull/2) Thank you


I have  made the corrections requested,  and I would like to know if there is anything else needed as I would like to work on another  Rnw file @jwokaty @villafup

**Comment by devbird007** [comment]
Hello @jwokaty 

This is my pull request here: https://github.com/Bioconductor/RBGL/pull/3

I also made sure to review it myself with the checklist and ensure it builds successfully.

**Comment by sonali8434** [comment]
> @jwokaty Please assign me GOstats/vignettes/GOvis.Rnw from project board. Thank you.

Hello @jwokaty this is my pull request for the above stated .Rnw to .Rmd conversion. 
https://github.com/Bioconductor/GOstats/pull/3

**Comment by sonali8434** [comment]
> @jwokaty Hi I would like to work on this vignette: chipseq/vignettes/Workflow.Rnw. Please assign me. Thank you

@Bukolab99 I think this vignette is already assigned to someone else you have to choose an unassigned vignette from project board https://github.com/orgs/Bioconductor/projects/2/views/6

**Comment by Bukolab99** [comment]
> > @jwokaty Hi I would like to work on this vignette: chipseq/vignettes/Workflow.Rnw. Please assign me. Thank you
> 
> @Bukolab99 I think this vignette is already assigned to someone else you have to choose an unassigned vignette from project board https://github.com/orgs/Bioconductor/projects/2/views/6

![Screenshot (28)](https://user-images.githubusercontent.com/112485492/226607632-9b7b07d1-a851-487d-8f86-205012fe61dd.png)
 It is unassigned. Check the first on the list

**Comment by sonali8434** [comment]
> > > @jwokaty Hi I would like to work on this vignette: chipseq/vignettes/Workflow.Rnw. Please assign me. Thank you
> > 
> > 
> > @Bukolab99 I think this vignette is already assigned to someone else you have to choose an unassigned vignette from project board https://github.com/orgs/Bioconductor/projects/2/views/6
> 
> ![Screenshot (28)](https://user-images.githubusercontent.com/112485492/226607632-9b7b07d1-a851-487d-8f86-205012fe61dd.png) It is unassigned. Check the first on the list
Yeah I have seen that sorry. That's why I have deleted my message soon . But you have already seen it . Once again sorry and good luck for your contribution.

**Comment by sopeadeniji** [comment]
@jwokaty  Please, review my pull request at https://github.com/Bioconductor/geneplotter/pull/3 . Thanks

**Comment by sonali8434** [comment]
> > @jwokaty Please assign me GOstats/vignettes/GOvis.Rnw from project board. Thank you.
> 
> Hello @jwokaty this is my pull request for the above stated .Rnw to .Rmd conversion. [Bioconductor/GOstats#3](https://github.com/Bioconductor/GOstats/pull/3)

Please add this pull request in the project board.

**Comment by jwokaty** [comment]
@sopeadeniji @sonali8434 I've updated the board with your PRs.

**Comment by jwokaty** [comment]
@kemicky I think it might be better to fork the repo, clone it, then copy your files over from your file-Rmd repo to your clone. Let's move this conversation to the sweave2rmd channel on slack. Please message me there.

**Comment by kemicky** [comment]
> @kemicky I think it might be better to fork the repo, clone it, then copy your files over from your file-Rmd repo to your clone. Let's move this conversation to the sweave2rmd channel on slack. Please message me there.

Ok, thank you I have done as you instructed and explained, and made the right pull requestor the GitHub repo for review: https://github.com/Bioconductor/Category/pull/4
thank you all

**Comment by devbird007** [comment]
Hi @villafup, on the project board, it appears that you have been assigned my PR. Here is the link to it: https://github.com/Bioconductor/RBGL/pull/3

I look forward to working with you on it.

**Comment by Dbn-tech** [comment]
@jwokaty .. i would like to work on this last unassigned Pr  OrganismDbi/vignettes/OrganismDbi.Rnw(https://github.com/Bioconductor/OrganismDbi) thank you

**Comment by devbird007** [comment]
> @jwokaty .. i would like to work on this last unassigned Pr OrganismDbi/vignettes/OrganismDbi.Rnw(https://github.com/Bioconductor/OrganismDbi) thank you

Hello @Dbn-tech , please see Jen's comment [here](https://github.com/Bioconductor/sweave2rmd/issues/63#issuecomment-1460283900) and suggestion #3 of another comment [here](https://github.com/Bioconductor/sweave2rmd/issues/63#issuecomment-1460834595) where they clarify that your PR would need to complete the review process and be merged before you can be assigned another vignette to work on.

**Comment by sirjoka** [comment]
Please can I get this vignette OrganismDbi/vignettes/OrganismDbi.Rnw assigned to me

**Comment by jwokaty** [comment]
@Dbn-tech Your first PR must be merged in order to work on another vignette.

@Bukolab99 I've put your PR on the board.

@sirjoka I can assign it to you.

**Comment by Khadeeejah** [comment]
@jwokaty  is it possible to work on a vignette that has been assigned to someone else but the status has been on "in progress" long ago ?

As there are no more Vignettes to work on .

**Comment by jwokaty** [comment]
@Khadeeejah I'm hesitant to allow that as they could be working on those vignettes. In the end, I can only merge 1 PR.

I am trying to find more vignettes; however, I have to ask if the maintainer will accept the PRs. I will update the board as I have more and notify here when they are available.

**Comment by Khadeeejah** [comment]
> @Khadeeejah I'm hesitant to allow that as they could be working on those vignettes. In the end, I can only merge 1 PR.
> 
> 
> 
> I am trying to find more vignettes; however, I have to ask if the maintainer will accept the PRs. I will update the board as I have more and notify here when they are available.

Noted thank you

**Comment by kemicky** [comment]
@jwokaty : Good day all, I am almost done with mine. I will PR it by today

**Comment by kemicky** [comment]
Hello colleagues,  some of us are newbies and we are doing our best to get this right. Everyone is doing their best and some of us it takes us all nights and lots of doing this over and over again to get it right . Can you give us a chance and why would anyone want to work on something someone had already done half way cos definitely the person who worked on it that much must have solved some of the work, why want to take over someones else hard work ?!. Please be patient. I am finishing it up.  Thank you

**Comment by Khadeeejah** [comment]
> Hello colleagues,  some of us are newbies and we are doing our best to get this right. Everyone is doing their best and some of us it takes us all nights and lots of doing this over and over again to get it right . Can you give us a chance and why would anyone want to work on something someone had already done half way cos definitely the person who worked on it that much must have solved some of the work, why want to take over someones else hard work ?!. Please be patient. I am finishing it up.  Thank you

Hi based on what I can see on the board. You have a PR already, your task doesn't fall in that category.

I was only referring to issues that do not have prs attached to it and they have been assigned to someone since the beginning of the contribution phase.

You can refer back to my message for clarity.

**Comment by kemicky** [comment]
> > Hello colleagues,  some of us are newbies and we are doing our best to get this right. Everyone is doing their best and some of us it takes us all nights and lots of doing this over and over again to get it right . Can you give us a chance and why would anyone want to work on something someone had already done half way cos definitely the person who worked on it that much must have solved some of the work, why want to take over someones else hard work ?!. Please be patient. I am finishing it up.  Thank you
> 
> Hi based on what I can see on the board. You have a PR already, your task doesn't fall in that category.
> 
> I was only referring to issues that do not have prs attached to it and they have been assigned to someone since the beginning of the contribution phase.
> 
> You can refer back to my message for clarity.

Ok, thank you for clarity.

**Comment by sopeadeniji** [comment]
@jwokaty I have updated visualize.Rmd @ https://github.com/Bioconductor/geneplotter/pull/3 based on your revisions. Thanks.

**Comment by Khadeeejah** [comment]
@jwokaty  I just saw this on the board

https://github.com/mtmorgan/DirichletMultinomial/blob/master/vignettes/DirichletMultinomial.Rnw

Kindly assign it to me 
Thank you

**Comment by kemicky** [comment]
@jwokaty 
please I see all the vignettes are assigned , please may you assign  another unassigned vignettes me thank you.

**Comment by sirjoka** [comment]
@jwokaty Here is my pull request for the OrganismDbi sweave conversion.
https://github.com/Bioconductor/OrganismDbi/pull/3

Please have a look when free. Thank you

**Comment by sopeadeniji** [comment]
@jwokaty visualize.Rmd updated @ https://github.com/Bioconductor/geneplotter/pull/3. Thanks.

**Comment by bellatrick** [comment]
@jwokaty kindly assign GeneMeta/vignettes/GeneMeta.Rnw to me. I'd like to work on it

**Comment by Dbn-tech** [comment]
> Hello, @jwokaty @mcarlsn I would like to work on this last unassigned Pr OrganismDbi/vignettes/OrganismDbi.Rnw(https://github.com/Bioconductor/OrganismDbi) thank you

hello, @jwokaty @villafup, i would like to know whether there are any other changes needed on my PR kindy..I would like to work on another Rnw after that one is merged

**Comment by sonali8434** [comment]
@jwokaty I want to work on this ShortRead/vignettes/Overview.Rnw

**Comment by sonali8434** [comment]
@jwokaty I want to work on this vignette ShortRead/vignettes/Overview.Rnw. Please assign me this.

**Comment by Dbn-tech** [comment]
Hello, @jwokaty @mcarlsn  can i work on this vignette please GeneMeta/vignettes/GeneMeta.Rnw. kindly at your free time assign it to me
